eZ/Bundle/EzPublishCoreBundle/Command/DeleteContentTranslationCommand.php

```diff
@@ -173,7 +173,7 @@ function ($versionLanguageCode) use ($languageCode) {
         if (empty($mainTranslationCandidates)) {
             throw new InvalidArgumentException(
                 'language-code',
-                "The last Version of the Content {$contentName} has no other Translations beside the main one"
+                "The last version of Content item {$contentName} has no other translations beside the main one"
```

Comment:
we do capitalize Content, even if it is a plain english word, because it is a specific "entity" in eZ. I think Version and Translation could also be kept capitalized, as they refer to specific entities as well...